content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -214,7 +214,7 @@ const Stop: FC<{
                 properties: ['margin-bottom'],
               }}
             >
-              <Divider color="warmNeutral.400" isKeyline={true} />
+              <Divider color="warmNeutral.400" />
```

Comment:
I think David meant to whole thing, it won't display without `isKeyline` or `isThin`, right? Maybe the required margin can be dealt with differently?

Reply:
You're right – that is what I meant

Reply:
Ok, it is completely gone now